_posts/2024-06-06-strimzicon2024-roundup.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title:  "StrimziCon 2024: Event Roundup and Session Recordings"
```

Comment:
amybe we could have a more catchy title ... "StrimziCon 2024: it was a blast!" or something around that, you will come with ideas better than mine for sure :-)

Reply:
I wanted to get something in around the recordings because I think people will search for that, but I'm rubbish at titles!

Reply:
Updated to something a bit more interesting